imports/ui_3/helpers/api-display.js

```diff
@@ -138,6 +140,11 @@ Template.registerHelper('displayKey', function displayKey(key) {
 });
 
 Template.registerHelper('displayValue', function displayValue(key, value) {
+  if (key.includes('Cost')) {
+    const community = getActiveCommunity();
+    numeral.language(community.settings.language);
```

Comment:
A kliensen a numeral-t mar automatikusan raallitjuk egy autorun-ban az active community-nek a language-ere. Szoval ez itt felesleges.
Az email-t a szerverrol kuldjuk, ott nincs "active community", ezert ott be kell allitani kezzel minden email kuldeskor.

Reply:
Emailnel nem is ezt a helpert hasznaljuk, arra van egy kulon emailes displayValue helper.

Reply:
Igen, ezert irom hogy abban tenyleg szukseg van a numeral beallitasara, de ebben nincs

Reply:
Kivettem belole, es behuztam a PR-t